### PR TITLE
Propose apk

### PR DESCRIPTION
**Pull request by Foxboron**
Reviewed-by: Ariadne Conill <ariadne@dereferenced.org>
Signed-off-by: Morten Linderud <morten@linderud.pw>

### CODE REVIEW

File: PURL-TYPES.rst [hunk 1]
```diff
@@ -23,6 +23,23 @@ See also https://github.com/package-url/purl-spec and
...
 ~~~~~~~~~~~~~~~~~~~~
 
+apk
+---
+``apk`` for APK-based packages:
+
+- There is no default package repository: this should be implied either from
+  the ``distro`` qualifiers key  or using a repository base url as 
+  ``repository_url`` qualifiers key.
+- The ``namespace`` is the vendor such as ``alpine`` or ``openwrt``.
```
Comment (stevespringett) [inline PURL-TYPES.rst:33]: If we compare the namespace of the "rpm" purl type, rpm states that "It is not case sensitive and must be lowercased.". What is the criteria for the apk namespace?

**Review by stevespringett — COMMENTED** [review]
Other than the clarification of the namespace case sensitivity, it looks good to me.

File: PURL-TYPES.rst (hunk 1 above)
Comment (Foxboron) [reply PURL-TYPES.rst]: It's largely similar to how I structured the spec for ALPM. I somehow forgot the line so I've added it back into the section. I'm not aware of any other namespace criterias.

**Review by wagoodman — APPROVED** [review]
🚀

**Comment by garethr** [comment]
Aside from the case sensitivity point above, anything else blocking this being merged? I'm happy to finish things off as this looks like it's stalled.

**Comment by Foxboron** [comment]
All mentioned issues has been fixed. I'm not sure why this hasn't been merged yet.

**Review by stevespringett — APPROVED** [review]
LGTM

**Comment by stevespringett** [comment]
Please resolve the conflicts so this can be merged.

**Comment by Foxboron** [comment]
@stevespringett rebased and should be all good now

**Review by stevespringett — APPROVED** [review]

**Pull request merged**